src/utils.js

```diff
@@ -37,11 +34,7 @@ export const WIDGET_SELECTED_CLASS_NAME = 'ck-widget_selected';
  * @returns {Boolean}
  */
 export function isWidget( node ) {
-	if ( !node.is( 'element' ) ) {
-		return false;
-	}
-
-	return !!node.getCustomProperty( widgetSymbol );
+	return node.is( 'element' ) && node.hasClass( WIDGET_CLASS_NAME );
```

Comment:
It needs to be a container element, am I right? I don't think any other element type would work. cc @pjasiun

Reply:
If that's true, I'd add a `is( 'containerElement' )` check also inside `toWidget()`. And similar one for `toWidgetEditable()` for `is( 'editableElement' )`